### ISSUE TITLE
Can ib_async support OCO order

### ISSUE THREAD

**Issue by lukeymyuan** [ib-api-reloaded/ib_async#178]
One-Cancels-Another (OCA) Order
https://www.interactivebrokers.com/campus/trading-lessons/using-the-one-cancels-another-oca-order-attribute-in-ibkrs-ibusopt/

**Comment by August1328** [comment]
Yes, ib_async supoorts OCA orders. For example, it is implemented in the `BracketOrder()` function.

Or, if you create orders without using ib_async functionality, you need to make sure to link the take-proft or stop-loss or whatever order to the initial parent order, like this:

```parent = Order()
parent.action = 'BUY'
parent.orderType = ...
parent.auxPrice = ...
parent.totalQuantity = ...
parent.orderId = ib.client.getReqId()
parent.transmit = False

stop_loss = Order()
stop_loss.action = 'SELL'
stop_loss.orderType = ...
stop_loss.auxPrice = ...
stop_loss.totalQuantity = ...
stop_loss.parentId = parent.orderId
stop_loss.orderId = ib.client.getReqId()
stop_loss.transmit = True

ib.placeOrder(contract, parent)
ib.placeOrder(contract, stop_loss)
ib.sleep(0.7)
```

The `ib.sleep()` at the end is necessary. The duration might be a bit long, but I never had a problem with orders not getting placed...

**Comment by gnzsnz** [comment]
try with `ib.oneCancelsAll`
